invoicing/ts-classes/Invoice.ts

```diff
@@ -0,0 +1,3 @@
+import {YInvoice} from "invoicing";
```

Comment:
@pcrispim what's the convention for the name of the base class ? and for the directory containing typescript classes ?

Reply:
defaults to prefix `Y` and pascal case

Reply:
There is no convention. We dictate it. The class is generated by https://github.com/YupanaInc/tools/pull/117/